src/v3/src/transformer/layout/idp/transformIdvIdpAuthenticator.ts

```diff
@@ -58,7 +94,36 @@ export const transformIdvIdpAuthenticator: IdxStepTransformer = ({
     },
   };
 
-  uischema.elements = [titleElement, descriptionElement, submitButton];
+  const divider: DividerElement = {
+    type: 'Divider',
+  };
+
+  uischema.elements = [
+    titleElement,
+    descriptionElement,
+    submitButton,
+    divider,
+  ];
+
+  if (shouldShowCancelLink(widgetProps?.features)) {
```

Comment:
Nice catch here to enforce the `shouldShowCancelLink()` logic. If you look at `transformCancelButton()` we also assert that the transaction has `cancel` [as an available step](https://github.com/okta/okta-signin-widget/blob/8c660a4277d5113e81efe9c9a70cdeaedde32955/src/v3/src/transformer/button/transformCancelButton.ts#L33-L34). 

Could you add that same assertion here before hardcoding in `step: 'cancel'`

Reply:
In persona it will always be `cancel`. So no need of additional check. Link will be shown/hidden only based on `shouldShowCancelLink`. 
where as `transformCancelButton` is a common code used in multiple places so they had to do additional check.